src/app/room/components/room.component.html

```diff
@@ -4,9 +4,8 @@
   <player [player]="firebasePlayer"></player>
 </div>
 <choice></choice>
-
 <div class="top-spacer">
-  <opponents [hasCurrentPlayerChosen]="(firebasePlayer$ | async)?.choice !== undefined"></opponents>
+  <opponents [hasCurrentPlayerChosen]="(firebasePlayer$ | async)?.choice !== null && (firebasePlayer$ | async)?.choice !== undefined"></opponents>
```

Comment:
How about double banging this and catching every falsy value?

```suggestion
  <opponents [hasCurrentPlayerChosen]="!!(firebasePlayer$ | async)?.choice"></opponents>
```

Reply:
love it